data/free worlds epilogue.txt

```diff
@@ -177,6 +183,12 @@ mission "FW Epilogue: Danforth"
 			label end
 			`	"Are the pirates giving you any trouble?" you ask.`
 			`	"A bit," he says. "We still have far too many pirates in this sector, in part due to the Free Worlds driving them out of the south. The pirates grew stronger and more bold as the war dragged on, when all our fleets were busy elsewhere, but now we're beginning to get them under control again."`
+			choice checkmate2
+				has "free worlds checkmate"
 			`	Then he leans forward and says, more quietly, "Of course, you and I both know the pirates aren't the main threat. Raven is off infiltrating Syndicate worlds even as we speak, and several of my other best intelligence officers, too. The Syndicate wants us to believe that they've weeded out all the rotten apples from their ranks, but we're keeping an eye on them, all the same."`
+				goto end
+			label checkmate2
+			`	Then he leans forward and says, more quietly, "Just a heads up that the Navy has their eye on the Free Worlds still. Just because a peace treaty was made doesn't mean you can keep your nukes forever. We destroyed all the means of making nukes for the terrorists on Buccaneer Bay for a reason."`
```

Comment:
Looks like I still have mention of #4672 in this bit here. I can't really remove this without making this paragraph super short, so I'll just leave it in and assuming that #4672 will also be merged.